web/libs/datamanager/src/components/MainView/GridView/GridView.scss

```diff
@@ -52,7 +52,7 @@
         display: grid;
         grid-auto-flow: row;
         grid-template-rows: minmax(1rem, 1fr);
-      grid-template-columns: 100%;
```

Comment:
im surprised this change made it to develop - wouldnt have this triggered a lint error?

Reply:
apparently, no